e2e/app/google-login.ts

```diff
@@ -37,7 +37,7 @@ export default class GoogleLoginPage extends BasePage {
    * Google login button.
    */
   async loginButton(): Promise<ElementHandle> {
-    return await this.page.waitForXPath(selectors.loginButton, {visible: true});
+    return await this.page.waitForXPath(selectors.loginButton, {visible: true, timeout: 10000});
```

Comment:
shorten timeout from default 30 sec to 10 sec. no use to waste time to wait if it is not there.